documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -628,6 +640,48 @@ NOTE: This list is not exhaustive - there may be codes in use that are not in th
 | 8GB
 | Sony UK
 
+| b04180
+| CM5
+| 1.0
+| 2GB
+| Sony UK
+
+| c04180
+| CM5
+| 1.0
+| 4GB
+| Sony UK
+
+| d04180
+| CM5
+| 1.0
+| 8GB
+| Sony UK
+
+| d04190
+| 500
+| 1.0
+| 8GB
+| Sony UK
+
+| b041a0
+| CM5 Lite
+| 1.0
+| 2GB
+| Sony UK
+
+| c041a0
+| CM5 Lite
+| 1.0
+| 4GB
+| Sony UK
+
+| d041a0
+| CM5 Lite
```

Comment:
Same for this chart. Unless the other Lite CMs don't have different revision codes?

Reply:
Previous CMs had the same revision code for LITE / EMMC

Reply:
Correct. Looks like CM5 is the first time that there's been a different revision code for the CM and CM-Lite variants.
Perhaps this also means that CM5 Lite will have its own entry in https://www.raspberrypi.com/documentation/computers/raspberry-pi.html#check-raspberry-pi-model-and-cpu-across-distributions ?